### PR TITLE
1.19-pre4 stuff

### PR DESCRIPTION
**Pull request by apple502j**
Expired messages on the server no longer get discarded - javadoc updated to reflect this change. And mapped several trivial method names and one new field.

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/ingame/HandledScreen.mapping [hunk 1]
```diff
@@ -95,3 +95,4 @@ CLASS net/minecraft/class_465 net/minecraft/client/gui/screen/ingame/HandledScre
 		ARG 2 y
 		ARG 3 z
 	METHOD method_37432 handledScreenTick ()V
+	METHOD method_44339 clearTouchDragStack ()V
```
Comment (liach) [inline mappings/net/minecraft/client/gui/screen/ingame/HandledScreen.mapping]: Is this more like
```suggestion
	METHOD method_44339 endTouchDrag ()V
```

File: mappings/net/minecraft/network/packet/c2s/play/ChatMessageC2SPacket.mapping [hunk 2]
```diff
@@ -10,8 +10,8 @@ CLASS net/minecraft/class_2797 net/minecraft/network/packet/c2s/play/ChatMessage
 	COMMENT reject the message and disconnect the client.
 	COMMENT
 	COMMENT <p>Messages that took more than {@link #TIME_TO_LIVE} to reach the server
-	COMMENT are considered expired and will be discarded. Messages can also be discarded
-	COMMENT if the server receives them with improper order.
+	COMMENT are considered expired and log warnings on the server.. Messages will be
```
Comment (liach) [inline mappings/net/minecraft/network/packet/c2s/play/ChatMessageC2SPacket.mapping]:
```suggestion
	COMMENT are considered expired and log warnings on the server. Messages will be
```

File: mappings/net/minecraft/network/packet/c2s/play/ChatMessageC2SPacket.mapping [hunk 3]
```diff
@@ -10,8 +10,8 @@ CLASS net/minecraft/class_2797 net/minecraft/network/packet/c2s/play/ChatMessage
...
+	COMMENT discarded if the server receives them with improper order.
```
Comment (liach) [inline mappings/net/minecraft/network/packet/c2s/play/ChatMessageC2SPacket.mapping]:
```suggestion
	COMMENT discarded if the server receives them in improper order.
```

File: mappings/net/minecraft/network/packet/c2s/play/CommandExecutionC2SPacket.mapping [hunk 4]
```diff
@@ -10,8 +10,8 @@ CLASS net/minecraft/class_7472 net/minecraft/network/packet/c2s/play/CommandExec
 	COMMENT reject the command and disconnect the client.
 	COMMENT
 	COMMENT <p>Commands that took more than {@link ChatMessageC2SPacket#TIME_TO_LIVE} to reach
-	COMMENT the server are considered expired and will be discarded. Commands can also be discarded
-	COMMENT if the server receives them with improper order.
+	COMMENT the server are considered expired and log warnings on the server. Commands will be
+	COMMENT discarded if the server receives them with improper order.
```
Comment (liach) [inline mappings/net/minecraft/network/packet/c2s/play/CommandExecutionC2SPacket.mapping]:
```suggestion
	COMMENT discarded if the server receives them in improper order.
```

File: mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping [hunk 5]
```diff
@@ -163,18 +163,18 @@ CLASS net/minecraft/class_3244 net/minecraft/server/network/ServerPlayNetworkHan
...
 		COMMENT @see ChatMessageC2SPacket#TIME_TO_LIVE
 		ARG 1 timestamp
 	METHOD method_44158 (Lnet/minecraft/class_2561;Lnet/minecraft/class_2561;)Lnet/minecraft/class_2561;
 		ARG 1 decorated
 	METHOD method_44159 decorate (Ljava/lang/String;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 query
-	METHOD method_44160 isDisordered (Ljava/time/Instant;)Z
-		COMMENT {@return whether the message sent at {@code timestamp} is received with improper order}
+	METHOD method_44160 isInProperOrder (Ljava/time/Instant;)Z
+		COMMENT {@return whether the message sent at {@code timestamp} is received with proper order}
```
Comment (liach) [inline mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping]:
```suggestion
		COMMENT {@return whether the message sent at {@code timestamp} is received in proper order}
```

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping [hunk 6]
```diff
@@ -183,6 +183,15 @@ CLASS net/minecraft/class_3244 net/minecraft/server/network/ServerPlayNetworkHan
 	METHOD method_44318 sendChatPreviewPacket (ILnet/minecraft/class_2561;)V
 		ARG 1 queryId
 		ARG 2 preview
+	METHOD method_44337 canAcceptMessage (Ljava/lang/String;Ljava/time/Instant;)Z
+		COMMENT {@return whether {@code message}, sent at {@code timestamp}, should be accepted}
+		COMMENT
+		COMMENT @implNote This returns {@code false} if the message is arrived with {@linkplain
```
Comment (Juuxel) [inline mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping]:
```suggestion
		COMMENT @implNote This returns {@code false} if the message arrives with {@linkplain
```
The original phrasing sounds a bit odd.

Also possible "with ... order" => "in ... order" here as well?

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**